pip/flatpak-pip-generator

```diff
@@ -139,12 +139,12 @@ def fprint(string: str) -> None:
 
 
 packages = []
-if opts.requirements_file and os.path.exists(opts.requirements_file):
-    with open(opts.requirements_file, 'r') as req_file:
+if opts.requirements_file and os.path.exists(os.path.expanduser(opts.requirements_file)):
```

Comment:
To avoid repeating ourselves, just store this in an `expanded_requirements_file` variable and use that throughout.